opencomputeproject/SAI
# Added ECMP LAG test for sail3.py

For sai server, testing different lags with router
        Steps
        1. Create virtual router, lag with rif and rif for src port
        2. add ports to the lag and the rest of the ports connect to rifs
        3. configure neighbors, nhops for all of the rifs
        4. make ecmp route with all of the nhops
        5. send packets from src port
        6. check polarization check in the lag and in the ecmp
        7. remove rifs, neighbors, nhops, lag members and route
        8. repeat steps 3-7 with differnt numbers of lag members and rifs
        8. clean up.
        """
